tools/docker/docker-compose-dev.yaml

```diff
@@ -25,6 +25,8 @@ x-environment: &common-env
   EDA_RULEBOOK_WORKER_QUEUES: "activation-node1,activation-node2"
   EDA_ANSIBLE_BASE_JWT_VALIDATE_CERT: ${EDA_ANSIBLE_BASE_JWT_VALIDATE_CERT:-False}
   EDA_ANSIBLE_BASE_JWT_KEY: ${EDA_ANSIBLE_BASE_JWT_KEY:-'https://localhost'}
+  EDA_RULEBOOK_LIVENESS_CHECK_SECONDS: ${EDA_RULEBOOK_LIVENESS_CHECK_SECONDS:-10}
+  EDA_RULEBOOK_LIVENESS_TIMEOUT_SECONDS: ${EDA_RULEBOOK_LIVENESS_TIMEOUT_SECONDS:-5}
```

Comment:
This inverts the default relationship between EDA_RULEBOOK_LIVENESS_CHECK_SECONDS and EDA_RULEBOOK_LIVENESS_TIMEOUT_SECONDS where the latter is larger than the former if neither are specified.

Reply:
What it's your point, anyway? The default values are intended for production and and provide some protection in cases with high workload. That's not the case of the development environment, where we want to detect it quickly and we don't need high timeouts.

Reply:
It's not about the values; smaller ones for development is reasonable.
It's that the default relation between the two is opposite the default implemented for deployment and is thus "odd" without explanation.  A comment explaining would benefit those, particularly the unfamiliar, who recognize the oddity and are left wondering "Why?"